services/api-server/src/simcore_service_api_server/utils/http_calls_capture.py

```diff
@@ -15,30 +19,40 @@ class HttpApiCallCaptureModel(BaseModel):
     description: str
     method: Literal["GET", "PUT", "POST", "PATCH", "DELETE"]
     host: str
-    path: str
+    path: PathDescription | str
     query: str | None = None
     request_payload: dict[str, Any] | None = None
     response_body: dict[str, Any] | list | None = None
     status_code: HTTPStatus = Field(default=HTTPStatus.OK)
 
     @classmethod
     def create_from_response(
-        cls, response: httpx.Response, name: str, description: str = ""
+        cls,
+        response: httpx.Response,
+        name: str,
+        description: str = "",
+        enhance_from_openapi_specs: bool = True,  # backwards compatibility
```

Comment:
If you want to be backwards compatible then the default neesd to be `False`, right?

Reply:
yep, I would like to keep it like this, but I removed the comment